src/backend/expungeservice/config.py

```diff
@@ -18,6 +19,7 @@ class Production(object):
     DEBUG = False
     TESTING = False
     SQLALCHEMY_DATABASE_URI = os.getenv('DATABASE_URL')
+    SECRET_KEY = os.getenv('SECRET_KEY')
```

Comment:
At this point the JWT_SECRET_KEY is only used for symmetric encryption of oeci credentials. Could we use SECRET_KEY for that instead, and only have one key in the app config again?  Or would overloading the purpose of this secret key be considered bad practice?

Reply:
I just didn't bother thinking through the security implications of reusing the same secret key so I was being conservative. My guess is that it is fine.